src/common/system.rs

```diff
@@ -1848,6 +1848,7 @@ pub struct ProcessRefreshKind {
     environ: UpdateKind,
     cmd: UpdateKind,
     exe: UpdateKind,
+    thread: bool,
```

Comment:
I still think that by default, this value should be `true`. So `Default` trait implementation should be manually written.

Reply:
done